### ISSUE TITLE
19. Working Group Meeting

### ISSUE THREAD

**Issue by Simsso** [Simsso/NIPS-2018-Adversarial-Vision-Challenge#91]
Date: 28. Oct 2018

### Work Items
* [x] Resolve #90, i.e. achieve ~~Tiny ImageNet validation accuracy > 70% with `avc-test-model`~~ not possible
* [x] Get submission results for LESCI (dependency: challenge backend robustness)
* [x] Compute correct classification ratio for projected / identity mapped samples
* [ ] Attack LESCI with non-gradient-based attacks from Foolbox (if such attacks exist)
* [ ] Evaluate and tune LESCI on Foolbox attacks #92 
* [x] Take care of PR #85 and merge it
* [x] Perform automated hyperparameter-search regarding `k` and `majority_threshold` for the LESCI ResNet
